scope.md

```diff
@@ -19,3 +21,20 @@ hyper-specific methods for one type of data to general, do-it-all packages
 (e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
 two. If you're interested in submitting your data visualization package, please 
 open a pre-submission inquiry on first.
+
+## Python package technical scope
+
+pyOpenSci is still defining the technical scope criteria for package review.  
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below. 
```

Comment:
I think this paragraph is a little confusing, and there are different mentions of scope here and in "aims and scope". Is it possible to unify those views? I would say that the package is out of scope if it fits one of those criteria + some other technical criteria defined here or elsewhere.

`Your package may not be in scope for us to review at this time if
it fits any of the criteria listed below:

**List of out-of-technical-scope criteria** 

Your python package should also comply with a list of technical criteria that is still being defined.`

Reply:
i totally can see how this is confusing. but this file is an include file. so it will be rendered on the aims and scope page. i just want to ensure you can see circle ci which does render the file for you. feel free to ask on slack and you can also ask @NickleDave as he is familiar with the ci setup too. i know you are diving in without some of the background context and appreciate the feedback!

Reply:
@arianesasso I ended up just building locally so I could see the page in context.  

One way to do this is to clone the repo, check out the pull request (e.g. with the GitHub cli you can do `gh pr checkout 148`) and then set up for dev as the README explains so you can run `nox -s docs-live` to get a live build in your local browser. For me this is easier than looking at artifacts on Circle CI.  

Reviewing a PR of a website is not easy compared to code! People must have other tools for this. I also tried the GitHub VS Code extension for reviewing PRs but I found it easier to just use the web interface

Reply:
awesome discussion y'all. thanks David for chiming in! building locally might be the easiest option. i don't use the github CLI. you can also grab instructions for git at the bottom of the pr to grab this branch from my fork!